### DISCUSSION TITLE
boost-vcpkg-helpers does not exist

### DISCUSSION THREAD

**Discussion by warsark** [microsoft/vcpkg#38545] [General]
VCPKG 2024-4-23
1.
git.exe pull
vcpkg update
2.
vcpkg upgrade --no-dry-run

error: The control or manifest file for boost-vcpkg-helpers could not be loaded due to the following error. Please remove boost-vcpkg-helpers and try again.
x:\vcpkg\ports\boost-vcpkg-helpers: error: boost-vcpkg-helpers does not exist

**Comment by Neumann-A** [comment]
Same answer as in https://github.com/microsoft/vcpkg/issues/38487#issuecomment-2085414942. Don't use `vcpkg upgrade`

**Reply by warsark** [reply]
when I use vcpkg x-set-installed
the vcpkg remove all installed package....
